### PR TITLE
Basic enter animations

### PR DESCRIPTION
**Pull request by aVileBroker**

### CODE REVIEW

**Comment by aVileBroker** [comment]
[Panel/Module/Card](https://trello.com/c/KtK45TzI/2-panel-module-card)

**Comment by aVileBroker** [comment]
[Text](https://trello.com/c/UhYSyfdk/1-text)

File: src/App.js [hunk 1]
```diff
@@ -45,7 +57,10 @@ class App extends Component {
     super(props);
 
     this.state = {
-      corruption: 0.2,
+      corruption: 0.1,
+      panels: [
+        { text: `${hacker.ingverb()} the ${hacker.noun()}...` },
```
Comment (nullbility) [inline src/App.js]: The hacker jargon string generation `${hacker.ingverb()} the ${hacker.noun()}...` should probably be a function

File: src/App.js [hunk 2]
```diff
@@ -22,9 +26,17 @@ const Plane = styled.div`
...
+  transform-style: preserve-3d;
+  transform: rotate3d(1, .5, -1, 30deg);
 `;
 
-/* transform: rotate3d(1, 1, 0, 10deg); */
+const PanelCSSGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 2rem;
+  grid-auto-rows: minmax(auto, auto);
```
Comment (nullbility) [inline src/App.js]: `user-select: none` may help

**Comment by aVileBroker** [comment]
👍

**Pull request merged**